test/lz4frame.jl

```diff
@@ -5,14 +5,15 @@
     version = LZ4.LZ4F_getVersion()
 
     @testset "Errors" begin
-        ERROR_GENERIC = UInt(18446744073709551615)
         no_error = UInt(0)
-
         @test !LZ4.LZ4F_isError(no_error)
         @test LZ4.LZ4F_getErrorName(no_error) == "Unspecified error code"
-
-        @test LZ4.LZ4F_isError(ERROR_GENERIC)
-        @test LZ4.LZ4F_getErrorName(ERROR_GENERIC) == "ERROR_GENERIC"
+        
+        if Sys.WORD_SIZE == 64
+            ERROR_GENERIC = UInt(18446744073709551615)
+            @test LZ4.LZ4F_isError(ERROR_GENERIC)
+            @test LZ4.LZ4F_getErrorName(ERROR_GENERIC) == "ERROR_GENERIC"
+        end
```

Comment:
Looking at this again should you just make `ERROR_GENERIC` a `UInt64`?

Reply:
...probably.

Reply:
On second thought probably not. The C code to define the error is using a `ptrdiff_t` into a list of error codes, the value of ERROR_GENERIC should be completely different.

Reply:
Ok, I think the standard way to check for this is `Int == Int64` as `Int` will change depending on if the system is 32-bit or 64-bit.

Reply:
Good to know. Thanks!